lib/auth/init.go

```diff
@@ -709,6 +710,21 @@ func generateAuthority(ctx context.Context, asrv *Server, caID types.CertAuthID)
 	return ca, nil
 }
 
+var secondFactorUpgradeInstructions = `
+Teleport v16 removed support for local authentication without a second
+factor enabled. The auth_service should be updated to enable it.
+
+auth_service:
+  authentication:
+    second_factor: on
+    webauthn:
+      rp_id: example.com
+
+For more information:
+- https://goteleport.com/docs/access-controls/guides/webauthn/
+- https://goteleport.com/docs/changelog/
```

Comment:
This link is only going to be helpful for a short period of time. After a while the part that mentions the second factor change will be too far down on the list and people won't notice it.

Should we just remove this link?